indent/crystal.vim

```diff
@@ -632,4 +632,4 @@ endfunction
 let &cpo = s:cpo_save
 unlet s:cpo_save
 
-" vim:set sw=2 sts=2 ts=8 et:
+set sw=2 sts=2 ts=8
```

Comment:
This is wrong, if you want to set the `shiftwitdh` option and some other, they should be in your own configs, not here!